omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml

```diff
@@ -637,16 +662,22 @@
 													<label>Condition of Baby at Discharge</label>
```

Comment:
use col-md-3 for this column and for the LLIN column above

Reply:
Done